.github/workflows/ci-test.yaml

```diff
@@ -17,46 +17,29 @@ jobs:
         ports:
           - 8545:8545
           - 8546:8546
-        env:
-          OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
-        options: >-
-          --rm
-          --health-cmd="test -f /CONTAINER_READY"
-          --health-start-period=90s
+        # env:
+        #   OASIS_DEPOSIT_BINARY: /oasis-deposit -test-mnemonic -n 5
+        # options: >-
+        #   --rm
+        #   --health-cmd="test -f /CONTAINER_READY"
+        #   --health-start-period=90s

```

Comment:
Check out here how to correctly configure the latest sapphire-localnet image with CI: https://docs.oasis.io/build/tools/localnet#github-actions

Reply:
Updated localnet config.